README.md

```diff
@@ -23,7 +23,7 @@ Note: older versions might not contain certain functionality (e.g. archival mock
 
 ## Default setup
 
-By running `docker-compose up -d` these steps take place: 
+By running `docker-compose up -d` these steps take place:
```

Comment:
I think we miss the new be here. I would add it there for now, and later we might rethink this README as the maintenance will become troublesome